scripts/visualise_results.py

```diff
@@ -8,11 +9,22 @@
 
 def plot_data(df):
     plt.style.use('seaborn-poster')
-    df.plot.line()
+
+    if (
+        0 < int(os.getenv("NUMBER_OF_DAYS", 0)) <= 45
+    ):  # To make the chart still easily digestible
+        df.plot.line(marker="o", markersize=8)
+        plt.grid(True, axis="both", alpha=0.3)
+    else:
+        df.plot.line()
+
+    plt.margins(0.03, 0.07)
+    plt.legend(frameon=False, prop={"size": 17})
     plt.xticks(df.index, df["DATE"], size="small", rotation=90)
     plt.yticks(size="small")
     plt.ylabel("Average Response Time (ms)")
     plt.xlabel("Run Date (DD-MM)")
+
```

Comment:
The xlabel is cut off.

Reply:
Fixed, also removed redundant margins and fixed date format.
![performance_graph](https://user-images.githubusercontent.com/35296336/90624146-68de3000-e20f-11ea-9da4-c85548bea5e1.png)